bundles/org.openhab.binding.groheondus/src/main/feature/feature.xml

```diff
@@ -4,9 +4,7 @@
 
     <feature name="openhab-binding-groheondus" description="GROHE ONDUS Binding" version="${project.version}">
         <feature>openhab-runtime-base</feature>
-        <bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-core/2.9.6</bundle>
-        <bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-annotations/2.9.6</bundle>
-        <bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-databind/2.9.6</bundle>
+        <feature dependency="true">openhab.tp-jackson</feature>
```

Comment:
Does this mean, that the dependency (ondus-api) will not work, if the library updates the version of jackson and openHAB didn't?

Reply:
That depends on the version. Usually incrementing micro version is binary-compatible, incrementing minor version could require re-compilation, major version is incompatible. So 2.8.1 and 2.8.9 should be compatible (mostly bugfix, smaller improvements) and 2.9.0 could be a problem. 

However, we found several problems when different versions of the same bundle are installed, so we try to reduce that.

Reply:
@FlorianSW As https://github.com/FlorianSW/grohe-ondus-api-java/pull/8 already has been merged we just need a new release of your lib and increase its version here too. Than there should be no problem anymore.

Reply:
See #5951.